jquery.scrolly.js

```diff
@@ -40,16 +40,11 @@
     this.velocity = this.$element.attr('data-velocity');
     this.bgStart = parseInt(this.$element.attr('data-fit'), 10);
 
-    $(document).scroll(function(){
+   $(document).on('scroll', function(){
       self.didScroll = true;
+      self.scrolly();
     });
-
-    setInterval(function() {
-      if (self.didScroll) {
-        self.didScroll = false;
-        self.scrolly();
-      }
-    }, 10);
```

Comment:
I did that on purpose. See http://joshbroton.com/hooking-up-to-the-window-onscroll-event-without-killing-your-performance/

The performance are still bad. If I want to re-write this plugin I'll use requestAnimationFrame which is way better https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame